assets/src/dashboard/components/cardGridItem/cardItemMenu.js

```diff
@@ -84,7 +84,7 @@ export default function CardItemMenu({
         className="grid-story-popover-menu"
         isOpen={contextMenuId === story.id}
         framelessButton
-        onSelect={onMenuItemSelected}
+        onSelect={(menuItem) => onMenuItemSelected(menuItem, story)}
```

Comment:
nit: could you update storybook to see story as well just so we keep things current there?

Reply:
Great call!